constants/roles.js

```diff
@@ -0,0 +1,6 @@
+const ROLES = {
+  SUPERUSER: 'superUser',
+  APPOWNER: 'appOwner'
+}
+
+module.exports = { ROLES }
```

Comment:
Why exporting as an object when `ROLES` itself is an object?
`module.exports = ROLES` should be a good option, and then in the routes, you can destructure and get `SUPERUSER` and `APPOWNER` values. Makes sense?